src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/NoFinalizerCheck.java

```diff
@@ -45,6 +45,22 @@
  * <pre>
  * &lt;module name=&quot;NoFinalizer&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ *  public class Test {
+ *
+ *      // Violation
```

Comment:
Please put `// Violation` at the end of line where the actual violation occurs.

Reply:
done